srcpkgs/pcsxr-pgxp/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'pcsxr-pgxp'
+pkgname=pcsxr-pgxp
+version=1.9.99
+revision=1
+lib32disabled=yes
+wrksrc="pcsxr-master"
+nostrip_files="pcsxr"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib
+ -DSND_BACKEND=sdl -DENABLE_CCDDA=ON -DUSE_LIBARCHIVE=ON -DUSE_LIBCDIO=ON -DOpenGL_GL_PREFERENCE=GLVND"
+hostmakedepends="pkg-config automake libtool intltool glib-devel gettext-devel nasm zip unzip"
```

Comment:
I understand `unzip` to extract the archive, but are you sure `zip` is needed too?

Reply:
See my comment below...